modern/clang/Dockerfile

```
@@ -161,4 +162,5 @@ RUN sudo mv /tmp/gcc/lib64 /usr/local/ \
     && sudo update-alternatives --install /usr/local/bin/ld ld /usr/local/bin/ld.lld 100 \
     && sudo rm /etc/ld.so.cache \
     && sudo ldconfig -C /etc/ld.so.cache \
-    && conan profile new --detect --force default
+    && sudo chmod +x /opt/conan/clang_bootstrap.sh \
+    && /opt/conan/clang_bootstrap.sh
```

Probably we don't want this file in the final image.

```suggestion
    && /opt/conan/clang_bootstrap.sh \
    && rm /opt/conan/clang_bootstrap.sh
```

---

done